standards/X509-SVID.md

```diff
@@ -92,7 +92,8 @@ In order to perform path validation, it is necessary to possess the public porti
 ### 5.2. Leaf Validation
 When authenticating a resource or caller, it is necessary to perform validation beyond what is covered by the X.509 standard. Namely, we must ensure that 1) the certificate is a leaf certificate, and 2) that the signing authority was authorized to issue it.
 
-When validating an X.509 SVID for authentication purposes, the validator MUST ensure that the `CA` field in the basic constraints extension is set to `false`, and that `keyCertSign` and `cRLSign` are not set in the key usage extension. The validator must also ensure that the scheme of the SPIFFE ID is set to `spiffe://`.
+When validating an X.509 SVID (leaf), the validator MUST ensure that the X509 version is 3 (encoded as 0x02). The validator MUST ensure
+that the `CA` field in the basic constraints extension, if present, is NOT set to `true`. The validator must also ensure that the scheme of the SPIFFE ID is set to `spiffe://`.
```

Comment:
Erroneous line break? The current doc doesn't constrain line length, though we can add breaks throughout if there is precedent for doing so